engines/bastion_katello/app/assets/javascripts/bastion_katello/sync-plans/details/views/sync-plan-details.html

```diff
@@ -49,5 +53,9 @@ <h2 class="fl" translate>Sync Plan {{ syncPlan.name }}</h2>
     </ul>
   </nav>
 
+  <div bst-alert="info" ng-show="task">
+    A sync has been initiated in the background, <a href="/foreman_tasks/tasks/{{ $scope.task.id }}">click for more details</a>
```

Comment:
for me when i did this, the task id did not get included in the URL.  It took me straight to the tasks list.